src/static/libs/js/auth.js

```diff
@@ -42,11 +21,7 @@ var options = {
   }]
 };
 
-// TODO: setup environment variables...
-var clientId = '',
-    domain = '';
-
-var lock = new Auth0Lock(clientId, domain, options);
+window.lock = new Auth0Lock(clientId, domain, options);
```

Comment:
Should 'work' but I suggest avoid having declaring anything on `window` to share objects.

If you need to reference this between files then perhaps a singleton should be exposed in another file and then `require()`d